Frameworks/Foundation/NSIndexPath.mm

```diff
@@ -18,6 +18,17 @@
 #import "StubReturn.h"
 #import "Foundation/NSIndexPath.h"
 
+#import <vector>
+#import <numeric>
+
+static NSString* s_archiveKey = @"_NS.IP";
```

Comment:
I've seen differing archiveKeys, went with this one since I was borrowing from IndexSet